src/reducers/email_flow_events/email-flows-event-reducer.js

```diff
@@ -27,6 +27,7 @@ export const DEFAULT_ENTITY = {
     event_type_name : '',
     summit_id : 0,
     recipient: '',
+    template_schema: null
```

Comment:
default isnt [] instad of null from api ? bc its a dictionay

Reply:
This prop is never an array. When it's populated is an object.